keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -715,30 +716,24 @@ to describe a selector based policy that applies to a multi-cluster service.
 
 _Optional, but recommended._
 
-MCS aims to align with the existing [service DNS
+The full specification for Multicluster Service DNS is in this KEP's [specification.md](specification.md). MCS aims to align with the existing [service DNS
 spec](https://github.com/kubernetes/dns/blob/master/docs/specification.md). This
-section assumes familiarity with in-cluster Service DNS behavior.
-```
-<<[UNRESOLVED]>>
-Full DNS spec needed prior to Beta graduation following:
-https://github.com/kubernetes/dns/blob/master/docs/specification.md
-<<[UNRESOLVED]>>
-```
-When a `ServiceExport` is created, this will cause a domain name for the
+section provides an overview of the multicluster DNS specification and its rationale, and assumes familiarity with in-cluster Service DNS behavior.
+
+In short, when a `ServiceExport` is created, this will cause a domain name for the
 multi-cluster service to become accessible from within the clusterset. The
-domain name will be `<service>.<ns>.svc.clusterset.local`. 
+domain name will be `<service>.<ns>.svc.clusterset.local`. This domain name operates differently depending on whether the `ServiceExport` refers to a ClusterSetIP or Headless service:
 
-**ClusterSetIP services:** Requests to this domain name from within an importing
+  * **ClusterSetIP services:** Requests to this domain name from within an importing
```

Comment:
should we use `<service>.<ns>.clusterset.cluster.local` to replace `<service>.<ns>.svc.cluster.local`? because `clusterset.local` is the cluster domain by `resolv.conf` and we can support search domain, then we can use the shorter domain name to do a request.

Reply:
> should we use <service>.<ns>.clusterset.cluster.local to replace <service>.<ns>.svc.cluster.local?

I think that ends up being a pretty confusing name - MCS is by definition not cluster.local and we probably don't want it to appear as such. That also has the impact of changing the default behavior of short-name lookups which is something the SIG has expressed some collective desire to avoid.

Reply:
Fixed the OP in [55fc3f6](https://github.com/kubernetes/enhancements/pull/2577/commits/55fc3f694c28a65ae39f3b685d4fdee95e670e92). Agree with Jeremy regarding the other thread so resolving here :)